### DISCUSSION TITLE
Implement log4j custom zeek scripts to detect cve-2021-44228 exploit attempts

### DISCUSSION THREAD

**Discussion by blarson1105** [Security-Onion-Solutions/securityonion#6683] [Unsupported Versions] [answered]
Team, 
Are you planning on pushing out Corelight's custom zeek package to aid detection efforts?  It would be amazing if this process were automated similar to **zkg install**.  

Also, do you have any thoughts on potential correlation searches to tie java processes to network traffic?  Perhaps with Sysmon? Is this a useful endeavor?

**Comment by dougburks** [comment, marked as answer]
https://blog.securityonion.net/2021/12/detecting-log4j-exploitation-attempts.html

**Reply by presianbg** [reply]
side question - in the example docs the custom zeek script is loaded via `/opt/so/saltstack/local/pillar/global.sls` , but so far I was using `local/pillar/zeek/init.sls` . Is there any difference ? Which one should be used ?

**Reply by dougburks** [reply]
`local/pillar/zeek/init.sls` was the original method for adding custom Zeek scripts. We later added support so that folks could more easily add custom Zeek scripts via the standard pillar files.

**Comment by blarson1105** [comment]
Thanks Doug.  Bryant was telling me you were losing your touch but I never
doubted you.   :).  Hope you all have a merry Christmas and thanks for all
you do.

**Reply by dougburks** [reply]
Merry Christmas to you too @blarson1105 !
